README.md

```diff
@@ -16,11 +16,9 @@ To run a website reading from two content services (`blog-articles` and `scholar
     content_page:
         pages:
             blog_article:
-                handler: 'libero'
```

Comment:
I'm wondering whether the [`<meta>` of an item](https://github.com/libero/schemas/pull/14) could declare what type of content it is (currently 'libero' or 'jats', but could be something custom). Reading this would allow us to narrow down what's triggered, without having to query the content at run time.

Reply:
What does "querying the content at run time" entails? You can envision situations when multiple formats for the content are present; maybe it's a matter of content negotiation if we can define a `Content-Type` more precise than `application/xml`.

Reply:
> What does "querying the content at run time" entails?
 https://github.com/libero/browser/blob/a9b3a68c4af2876d2a0e4d8d843d24a66703fbb2/vendor-extra/JatsContentBundle/src/EventListener/ContentHeaderListener.php#L22-L28

So it always listens but only cares about JATS content. This will be a problem if the content contains both Libero XML _and_ JATS (possible?). If it were declared somewhere that Libero is the primary format in that case, listeners like this could be excluded.

Reply:
Should work as long as there is a clear priority between the two different listeners (otherwise one would overwrite/clash with the other?)